src/components/Location.spec.js

```diff
@@ -0,0 +1,11 @@
+
+    import {render, screen} from '@testing-library/react'
+    import Location from './Location'
+
+    describe('Location', () => {
+        it('renders', () => {
+            render(<Location />)
+            expect(screen.getByText('Location')).toBeInTheDocument()
+        })
+    })
```

Comment:
You need to add props to the `Location` component:
```suggestion
        it('renders', () => {
            render(<Location image=”/some/path" text="some text" />)
            expect(screen.getByText('some text')).toBeInTheDocument()
            .... check for img ...
        })
    })
```